src/configdialog.cpp

```diff
@@ -597,18 +597,23 @@ ConfigDialog::ConfigDialog(QWidget *parent): QDialog(parent,Qt::Dialog|Qt::Windo
     connect(ui.tbRevertPDF, SIGNAL(clicked()), this, SLOT(revertClicked()));
 
 	// limit dialog size
-	QRect screen = QGuiApplication::primaryScreen()->geometry();
+	QRect screen = QGuiApplication::primaryScreen()->availableGeometry();
 	if (!screen.isEmpty()) {
 		int nwidth = width(), nheight = height();
 		if (nwidth > screen.width()) nwidth = screen.width();
 		if (nheight > screen.height()) nheight = screen.height();
 		if (nwidth == width() && nheight == height()) return;
-		resize(nwidth, nheight);
-		move(frameGeometry().right() > screen.right() ? screen.left() : x(),
-		     frameGeometry().bottom() > screen.bottom() ? screen.left() : y());
+		resize(nwidth - 32, nheight - 32);
```

Comment:
numbers coming out of the blue which are supposed to work over different styles/OS don't give much confidence ...

Reply:
It's change from one numbers to another:
![image](https://user-images.githubusercontent.com/50521366/213888068-e76bc74d-8439-4259-bdef-9850b894927d.png)
Above ones were causing the issue.

Reply:
@gaplin nheight-32 is used because the window decoration has a large height mainly due to the title bar. So I wonder why you use nwidth-32. This leaves many space to the left and right of the dialog because the left and right decorations are typically thin lines.

We should not use magic numbers (i.e. 32) if possible. If we do so we could add an explanation as a comment:

``// subtract 32 because the window decoration has to be taken into account, but we can only estimate the values``

I tried show() as explained in the [qt forum](https://forum.qt.io/topic/41135/solved-qdesktopwindow-availablegeometry-not-excluding-taskbar-qt-5-2-windows-7). For me it did not work but maybe you can figure out.

Reply:
The -32 on width is for side taskbar. I didn't think about it in the way you desribed(title bar). 
I thought window as a whole was possibly misplaced in both axis.

I'll take a look.

Reply:
availableGeometry is designed to calculate the size of the difference between the screen area and the taskbar (have a look [here](https://doc.qt.io/qt-6/qscreen.html#availableGeometry-prop)).

Reply:
Ye, so the availableGeometry simply shows size of maximized window.

Following hint with show() method I got rid of this 32 const. I'll create another PR.

Reply:
New PR: #2868